pom.xml

```diff
@@ -26,6 +26,7 @@
   </parent>
   <groupId>org.apache.spark</groupId>
   <artifactId>spark-parent_2.11</artifactId>
+  <!-- When updating version make sure to update ./python/pyspark/version.py as well -->
```

Comment:
So during the binary release process the version information is automatically written out. If this is a sticking point for @davies though I can see about doing it (but I'll also change the init to not import it then).